commands/uploader.go

```diff
@@ -333,9 +364,9 @@ func (c *uploadContext) Await() {
 // given "endpoint". If no state has been explicitly set, an "unknown" state
 // will be returned instead.
 func getVerifyStateFor(endpoint lfsapi.Endpoint) verifyState {
-	key := strings.Join([]string{"lfs", endpoint.Url, "locksverify"}, ".")
+	uc := config.NewURLConfig(cfg.Git)
 
-	v, ok := cfg.Git.Get(key)
+	v, ok := uc.Get("lfs", endpoint.Url, "locksverify")
```

Comment:
Good call, I think I got this in https://github.com/git-lfs/git-lfs/pull/2162/commits/aaa09af5cc859085dc3fe3ee8f78a5078b968864.